openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -821,14 +829,19 @@ def post(self, request):
             self.retire_entitlement_support_detail(user)
 
             # Retire misc. models that may contain PII of this user
-            SoftwareSecurePhotoVerification.retire_user(user.id)
             PendingEmailChange.delete_by_user_value(user, field='user')
             UserOrgTag.delete_by_user_value(user, field='user')
 
             # Retire any objects linked to the user via their original email
             CourseEnrollmentAllowed.delete_by_user_value(original_email, field='email')
             UnregisteredLearnerCohortAssignments.delete_by_user_value(original_email, field='email')
 
+            # This signal allows code in higher points of LMS to retire the user as necessary
+            USER_RETIRE_LMS_CRITICAL.send(sender=self.__class__, user=user)
+
+            # TODO: Password Reset links - https://openedx.atlassian.net/browse/PLAT-2104
+            # TODO: Delete OAuth2 records - https://openedx.atlassian.net/browse/EDUCATOR-2703
```

Comment:
What's meant by these TODO comments?  They both link to completed tasks.

Reply:
I had removed these in a previous PR. Probably were added back in a rebase?

Reply:
Ugh yes, rebase shenanigans. I'll fix.